src/border_agent/border_agent.cpp

```diff
@@ -251,6 +254,13 @@ void BorderAgent::Start(void)
     UpdateMeshCopService();
 
     otBorderAgentSetEphemeralKeyCallback(mHost.GetInstance(), BorderAgent::HandleEpskcStateChanged, this);
+
+#if OTBR_ENABLE_FEATURE_FLAGS
+    FeatureFlagList featureFlagList;
+
+    SetEphemeralKeyEnabled(featureFlagList.enable_ephemeralkey());
+    otbrLogInfo("BA Epskc Feature has been %sabled in init", (featureFlagList.enable_ephemeralkey() ? "en" : "dis"));
```

Comment:
These seems to do the same as `ApplyFeatureFlagList()` method below? Should we call it instead?
```suggestion
    ApplyFeatureFlagList(fearetuFlagList);
```

Reply:
Thanks, init part has been removed.